pkg/apis/core/validation/validation.go

```diff
@@ -4444,31 +4444,15 @@ func ValidateNode(node *core.Node) field.ErrorList {
 // ValidateNodeResources is used to make sure a node has valid capacity and allocatable values.
 func ValidateNodeResources(node *core.Node) field.ErrorList {
 	allErrs := field.ErrorList{}
-	// Validate resource quantities in capacity.
```

Comment:
Why to remove this comment? It still looks correct even with the change.

Reply:
You are right, thanks!

Added it now :smile: